crates/llama-cpp-bindings/build.rs

```diff
@@ -26,9 +31,16 @@ fn main() {
     if cfg!(feature = "cuda") {
         config.define("LLAMA_CUBLAS", "ON");
         config.define("CMAKE_POSITION_INDEPENDENT_CODE", "ON");
-        println!("cargo:rustc-link-search=native=/usr/local/cuda/lib64");
+        if cfg!(target_os = "windows") {
+            let Ok(cuda_path) = env::var("CUDA_PATH") else {
+                panic!("CUDA_PATH is not set");
+            };
+            println!(r"cargo:rustc-link-search=native={}\lib\x64", cuda_path);
+        } else {
+            println!("cargo:rustc-link-search=native=/usr/local/cuda/lib64");
+            println!("cargo:rustc-link-lib=culibos");
```

Comment:
Turns out, even cuda `11.7` doesn't contain `culibos` for windows (one build failure: https://github.com/TabbyML/tabby/actions/runs/7158762484/job/19491219353)
So move it to `else` branch.